__tests__/app.test.js

```diff
@@ -155,4 +155,106 @@ describe("All requests", () => {
                 });
         });
     });
+    describe("POST /api/articles/:article_id/comments", () => {
+        test("201: should return 200 and insert/post to add new record to the comments table", () => {
+            return request(app)
+                .post("/api/articles/9/comments")
+                .expect(201)
+                .send({
+                    user_name: "butter_bridge",
+                    body: {
+                        article_id: 9,
+                        author: "butter_bridge",
+                        votes: 20,
+                        created_at: new Date(),
+                        body: "post request will going to be ok",
+                    },
```

Comment:
For this ticket body should not be an object here, what we should expect is body to be a string containing only the comment you would like to post.